src/pages/FreeClass/PositionChoice.tsx

```diff
@@ -99,7 +102,7 @@ export const PositionChoice: MainStackScreen<"PositionChoice"> = () => {
     <PageWrapper>
       <View style={{ paddingTop: 28 }}>
         <View style={{ paddingLeft: 28 }}>
-          <Title>Posizione attuale</Title>
+          <Title>{t("freeClass_exactPosition", { ns: "freeClass" })}</Title>
```

Comment:
anche qui puoi specificare il namespace direttamente in `useTranslate("freeClass")` come in altri file